vars.sample

```diff
@@ -50,10 +50,20 @@ ORACLE_TABLESPACES_DIR=/home/oracle/app/oracle/oradata
 # }}}
 
 # Only needed for ETL
+ETL_SCRIPTS=/path/to/transmart-data/samples/postgres
+export PATH=$ETL_SCRIPTS:$PATH
+
+# export KETTLE_HOME=$HOME ## kettle.properties file is expected in .kettle subdirectory of KETTLE_HOME
```

Comment:
I you're not using the "make approach", then why use transmart-data at all? You can just dump the files in tranSMART-ETL as usual. Manually managing the kettle home is a constant source of pain and people (including me) will get confused if their settings in vars are ignored.